aicsimageio/readers/czi_reader.py

```diff
@@ -600,11 +599,11 @@ def _stitch_tiles(
             if dim is DimensionNames.SpatialX:
                 arr_shape_list.append(mosaic_bbox.w)
             if dim is DimensionNames.Samples:
-                arr_shape_list.append(data_dims_shape[dim][1])
+                arr_shape_list.append(data_dims_shape[CZI_SAMPLES_DIM_CHAR][1])
 
         ans = None
         if isinstance(data, da.Array):
-            ans = da.empty(
+            ans = da.zeros(
```

Comment:
I changes this from empty because when I actually ran a `compute` against da.empty where no values were filled (i.e. because the mosaic isn't a perfect square / cuboid) then it results sys.maxsize I believe soooooo yea... zeros is better.